### DISCUSSION TITLE
Embeddable Widget for other sites

### DISCUSSION THREAD

**Discussion by shlomof-pyle** [GitbookIO/community#809] [Feature Requests]
function to have a widget linked to a published site that inherits that sites permission. would prioritize the search bar, and would have a mobile style layout. an additional added bonus would be ability to link 3rd party urls to automatically open the widget to a specific gitbook url

**Comment by addisonschultz** [comment]
Hi @shlomof-pyle ! Addison here from GitBook - we’re working on a few things related to this feature request :) 

Can you reach out to me at addison (@) gitbook.com with the Subject on “Embeddable GitBook”? 

Would love to chat more!

**Comment by James4David** [comment]
You can achieve this with a simple iframe-based widget that inherits your site’s permissions and supports deep links. I’ve used this setup on [Sportzfy](https://sportzfysapk.com/)
  where the Sportzfy app is provided — to embed docs and open specific GitBook URLs directly in a mobile-style layout. A native GitBook widget for this would make it much cleaner

**Reply by addisonschultz** [reply]
Hey @James4David ! We’re working on a native embeddable widget - and have something ready for you to test! Can you reach out to me at addison (@) gitbook.com with the Subject on “Embeddable GitBook”?

Will get you set up!

**Comment by aikoii** [comment]
Hi everyone! 👋
Just closing the loop here - this is now available via our Docs Embed. More details here: https://gitbook.com/docs/publishing-documentation/embedding

**Comment by JetFault** [comment]
Is there a way to offer the same embedding for mobile apps (either native or via react native)?

**Reply by aikoii** [reply]
Yes, this should be possible. I think the approach is via a WebView. For React, we have docs here: https://gitbook.com/docs/publishing-documentation/embedding/implementation/react
Just note that this is designed for React web/iframe usage
